app/javascript/vue/protocol/step_elements/text.vue

```diff
@@ -87,6 +90,14 @@
         this.element.attributes.orderable.name = data.data.attributes.name
         this.element.attributes.orderable.updated_at = data.data.attributes.updated_at
         this.$emit('update', this.element, true)
+      },
+      duplicateElement() {
```

Comment:
these duplicateElement methods seem to be always the same, could be extracted into mixin